src/style.css

```diff
@@ -1,3 +1,63 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
   margin: 0;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  flex-direction: row;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  color: black;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  line-height: 14px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: var(--main-color);
```

Comment:
If you add transition when on hover it will be better

Reply:
but transitions and animations is 5th lesson